flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/DefaultVertexParallelismInfoTest.java

```diff
@@ -41,15 +41,15 @@ public void parallelismInvalid() {
         assertThrows(
                 "parallelism is not in valid bounds",
                 IllegalArgumentException.class,
-                () -> new DefaultVertexParallelismInfo(-1, 1, ALWAYS_VALID));
+                () -> new DefaultVertexParallelismInfo(-2, 1, ALWAYS_VALID));
     }
 
     @Test
     public void maxParallelismInvalid() {
         assertThrows(
                 "max parallelism is not in valid bounds",
                 IllegalArgumentException.class,
-                () -> new DefaultVertexParallelismInfo(1, -1, ALWAYS_VALID));
+                () -> new DefaultVertexParallelismInfo(1, -2, ALWAYS_VALID));
     }
 
     @Test
```

Comment:
Can we have some tests for the modification of `DefaultVertexParallelismInfo`? e.g. `testSetParallelism`, `testCreateWithUnconfiguredParallelism`?

Reply:
I will add tests.